spec/spec_helper.rb

```diff
@@ -17,39 +10,38 @@ def self.cache
 end
 
 # ActiveRecord::Base.logger = Logger.new(STDOUT)
-ActiveRecord::Base.establish_connection(:adapter => "sqlite3", :database => ":memory:")
+ActiveRecord::Base.establish_connection adapter: 'sqlite3', database: ':memory:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.